bin/check-zpool.rb

```diff
@@ -34,10 +34,15 @@ def run
     else
       zpools = SensuPluginsZFS::ZFS.zpools
     end
+    # Looks odd, but we want to check things that can go critical first
```

Comment:
another option would be to check everything, store results in an array, and then do the alerting at the end. I am not necessarily opposed to this but that is another option.

Reply:
I was planning to be lazy, but I think in addition to being cleaner, it's possible a zpool could still "warn" on capacity and hide a critical capacity zpool down the line.

Reply:
cool, I will wait for an update then before merging.